app/controllers/users/registrations_controller.rb

```diff
@@ -1,30 +1,39 @@
 class Users::RegistrationsController < Devise::RegistrationsController
-
   before_action :load_paperclip_vars
 
   def avatar
     user = User.find_by_id(params[:id]) || current_user
     style = params[:style] || "icon_small"
+    # TODO: Maybe avatar should be an Asset, so it's methods could be used,
+    # e.g. presigned_url in this case
```

Comment:
I'm not sure presigned_url for avatar should be even used, it can be public anyway.